router/router.go

```diff
@@ -51,6 +51,8 @@ func setDocumentController(r *gin.Engine) {
 	r.POST("/documents/viewer", controllers.GetViewerDocuments)
 	r.PUT("/document/update/status", controllers.SetDocumentStatus)
 	r.POST("/document/histories", controllers.GetDocumentHistories)
+	r.DELETE("/document/delete", controllers.DeleteDocument)
```

Comment:
Given that the method is already set to `DELETE`, it might be more straightforward to have the route named simply `/document`. Typically, _CRUD_ operations align the route with the corresponding HTTP methods. For instance, the API we utilized to create new documents employs the `POST` method, also accessible at `/document`.

Reply:
Oh! I see. The HTTP `DELETE` method clearly specifies what this API does. I've addressed this.